content/questions/actions/question-109.md

```diff
@@ -0,0 +1,13 @@
+---
+question: "What is the purpose of the `timeout-minutes` keyword in a step?"
+archetype: "questions"
+title: "Question 109"
+draft: false
+---
+
+
+> https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes
+1. [x] it limits the execution time for individual commands within a step
```

Comment:
I can't find any source that it limites the execution time for "individual commands" within a step. 

All I see is that it limits the execution time of the **entire** step.

Do you have some source for that?

Reply:
Steps can have [multi line commands ](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsrun) and I'm pretty sure that the `timeout-minutes` is not for individual commands but for the step as a whole.

Reply:
Yes, you are right, I have formulated it wrongly. I have updated the answer